litmus-portal/frontend/src/components/Unimodal/styles.ts

```diff
@@ -0,0 +1,58 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  modalContainer: {
+    width: '70%',
+    padding: '1rem',
+    margin: '2rem auto',
+    background: theme.palette.common.white,
+    borderRadius: 3,
+    textAlign: 'center',
+    outline: 'none',
+  },
+
+  darkModalContainer: {
+    width: '70%',
+    padding: '1rem',
+    margin: '2rem auto',
+    background: '#161616',
```

Comment:
Use this from the theme

Reply:
I actually used the hex code from the `YamlEditor` component as the `VerifyCommit` component showed that inside a modal. Will change this to use `theme.palette.common.black`.